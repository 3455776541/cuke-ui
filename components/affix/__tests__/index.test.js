import React from "react";
import { render } from "enzyme";
import toJson from "enzyme-to-json";
import Affix from "../index";

describe("<Affix/>", () => {
	it("should render a <Affix/> components", () => {
		const wrapper = render(
			<div>
				<Affix>
					<div>affix</div>
				</Affix>
				<Affix offsetTop={200}>
					<div>affix</div>
				</Affix>
			</div>
		);
		expect(toJson(wrapper)).toMatchSnapshot();
	});
});
