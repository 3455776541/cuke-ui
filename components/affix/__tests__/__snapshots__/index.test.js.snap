// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Affix/> should render a <Affix/> components 1`] = `
<div>
  <div
    class="cuke-affix"
    style="top:0"
  >
    <div>
      affix
    </div>
  </div>
  <div
    class="cuke-affix"
    style="top:200px"
  >
    <div>
      affix
    </div>
  </div>
</div>
`;
